parameters/navigation/navscan.yaml

```diff
@@ -0,0 +1,8 @@
+frame_id: base_link
```

Comment:
We used to do this in map frame. I think using the `base_link` frame, will cause the cloud to move with the robot. Or will it be fixed to the frame at a certain timestamp?

Reply:
Not 100% sure. I thought it best to publish in base link so that we don't introduce errors with localisation.

Reply:
I do understand why you would do that. But this is very critical. When older scans would move with the base_link, you are introducing much bigger errors...

So we need to either check this in the code or test it on the robot.

Reply:
Tested: A small delay is indeed present when moving the robot. This can be solved in one of two ways. express the points in map frame so that it publishes the absolute points OR change the timestamp of the pointcloud message so that it uses the pose of base_link from when the image was received. The latter option is being implemented in https://github.com/tue-robotics/depthimage_to_navscan/pull/7